Troubleshooting follower reads on global tables (#15930)
cockroachdb/docs: v21.2/global-tables.md

For more information about how to choose a database survival goal, see [When to Use `ZONE` vs. `REGION` Survival Goals](when-to-use-zone-vs-region-survival-goals.html).

## Troubleshooting

### High follower read latency on global tables

> Reads on multiregion global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms. 

Reviewer: Just the comment that github added - change this from multiregion to multi-region

Author: thanks Andrew - pushed a fix!

> reads on multi-region global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms. 

Reviewer 2: ```suggestion
Reads on multi-region global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms. 
```

Author: Fixed, thanks!

> Note that this value will impact write latency to global tables, so you should proceed in 100ms increments until the high read latency no longer occurs. If you've increased the setting to 1500ms and the problem persists, you should [contact support](support-resources.html).

Reviewer 2: ```suggestion
The value of `kv.closed_timestamp.lead_for_global_reads_override` will impact write latency to global tables, so you should proceed in 100ms increments until the high read latency no longer occurs. If you've increased the setting to 1500ms and the problem persists, you should [contact support](support-resources.html).
```

Author: Fixed, thanks!
